README.md

```diff
@@ -18,24 +18,21 @@ This is a Ruby on Rails application for internal use by GDS and some departmenta
 
 It has these features:
 
-- Creating & editing topics and mainstream browse pages
+- Creating & editing topics and mainstream browse pages.
 - Curate the pages tagged to a topic or browse page into "curated lists". This means they'll be displayed in a group with a title. By default, the links are not curated and will appear with a 'A-Z' heading.
 
 ![Screenshot of curated and non-curated pages](docs/screenshot-curated-topics.png)
 
 Left, a [curated mainstream browse page](https://www.gov.uk/browse/childcare-parenting/childcare), right, an [uncurated browse page](https://www.gov.uk/browse/justice/prisons-probation).
 
-Collections publisher retrieves information about which content is tagged to a topic or browse page from [rummager](https://github.com/alphagov/rummager). It publishes the curated lists to the [publishing-api](https://github.com/alphagov/publishing-api).
```

Comment:
Could leave in the curated lists bit?

Reply:
@brenetic I took it out and explained below when describing the `publishing-api` dependency. Less repetition! Also the line below about sidekiq also makes it clear this app publishes to the publishing-api.